lib/Compilation.js

```diff
@@ -1473,12 +1877,15 @@ class Compilation extends Tapable {
 
 		const blocks = block.blocks;
 		for (let indexBlock = 0; indexBlock < blocks.length; indexBlock++) {
-			const chunks = blocks[indexBlock].chunks;
```

Comment:
@sokra this appears to be a bug given that .chunks is a getter that throws an error. 

I assume this Is supposed to be removing the connection. Between this deep block and a ChubkGroup (ignore my part of the diff which is like wrong)